app/javascript/convert.js

```diff
@@ -0,0 +1,38 @@
+import {saveAs} from "file-saver";
+
+$(() => {
+  $("form#convert").submit((evt) => {
+    evt.preventDefault();
+    let formData = new FormData();
+    formData.set("file", $("input[type=file")[0].files[0]);
+    formData.set("format", $("select").val());
+    formData.set("historic", $("input[type=checkbox]").is("checked"));
+    $("#dropzone-overlay").fadeTo(125, 0.9);
+    $("#droplabel").text("Converting file...");
+    window.isUploading = true;
+    window.showSpinner("#fff");
+
+    $.ajax({
+      method: "POST",
+      url: "/api/v4/convert",
+      data: formData,
+      cache: false,
+      processData: false,
+      contentType: false,
+      success: (response, textStatus, xhr) => {
+        window.isUploading = false;
+        window.spinner.stop();
+        let filename = xhr.getResponseHeader("X-Filename");
+        var blob = new Blob([response]);
+        saveAs(blob, filename);
+        $("#dropzone-overlay").fadeOut(125);
+      },
+      error: (xhr) => {
+        window.isUploading = false;
+        window.spinner.stop();
+        $("#droplabel").text(`Error converting: ${xhr.responseJSON.message}`);
+      }
+    });
+    return false;
```

Comment:
Why does this function return false?

Reply:
I mistakenly thought it was for preventing jQuery propogation, but it is essentially just evt.preventDefault and evt.stopPropogation.  I'll remove the return since it's not needed.